rp2040-hal/src/pio.rs

```diff
@@ -650,6 +616,54 @@ impl<SM: ValidStateMachine> StateMachine<SM, Running> {
     }
 }
 
+/// PIO RX FIFO handle.
+pub struct Rx<SM: ValidStateMachine> {
+    block: *const rp2040_pac::pio0::RegisterBlock,
+    _phantom: core::marker::PhantomData<SM>,
+}
+
+impl<SM: ValidStateMachine> Rx<SM> {
+    /// Get the next element from RX FIFO.
+    ///
+    /// Returns `None` if the FIFO is empty.
+    pub fn read_rx(&mut self) -> Option<u32> {
```

Comment:
Could this be just `read`? And `write` below?

Reply:
Changed.